mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -1,65 +1,101 @@
 pipeline {
-    agent {
-        docker {
-            image 'rocm/mlir:miopen_test'
-            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
-            label 'gfx908'
-        }
-    }
+    agent none
     stages {
-        stage('Environment') {
-            steps {
-                sh 'cat /etc/os-release'
-                sh '/opt/rocm/bin/rocm-smi'
-            }
-        }
-        stage('Build libMLIRMIOpen') {
-            steps {
-                checkout scm
-                sh '''
-                    mkdir -p build && cd build
-                    /usr/local/bin/cmake -G "Unix Makefiles" ../llvm \
-                         -DLLVM_ENABLE_PROJECTS="mlir;lld" \
-                         -DCMAKE_BUILD_TYPE=Release \
-                         -DBUILD_SHARED_LIBS=OFF \
-                         -DLLVM_BUILD_LLVM_DYLIB=OFF \
-                         -DLLVM_ENABLE_TERMINFO=OFF
-                    make -j$(nproc) libMLIRMIOpen
-                '''
+        stage("Carry out test") {
```

Comment:
[Question] I find a number of quite trivial logs for each matrix cells in your jenkins run # 12. Is it possible to remove those?

Another request is if it is possible to not linear-ize the parallel run of different stage cells. If multiple certain matrix cells has a failure in, for example, installation of a dependency. How do you tell which failure is matched to which run? This might have something to do with how jenkins pipeline is setup, maybe follow up with @okakarpa on how he is able to set up the pipeline in a graph manner. Refer to the CI run log [here](http://10.236.106.97:8080/blue/organizations/jenkins/mlir/detail/PR-215/1/pipeline) as an example.

Reply:
If you go to the Blue Ocean link, you'll get a more compact view of the situation - http://10.236.106.97:8080/blue/organizations/jenkins/MLLIR-MIopen-Resnet50-refactor-dev/detail/MLLIR-MIopen-Resnet50-refactor-dev/12/pipeline

Reply:
It looks like that big horizontal list is just how the stage view plugin works and there's nothing we can do about it

Reply:
@krzysz00 Awesome. I think blue ocean view is what I'm asking for.

Reply:
Please rename `Carry out test` to `Resnet50 configs tests`. It looks like the entire stages aggregation are called `Carry out test` now.